sw/device/tests/alert_handler_ping_timeout_test.c

```diff
@@ -68,11 +68,9 @@ static void init_peripherals(void) {
  * fires and check the local alert cause register.
  */
 static void alert_handler_config(void) {
+  // Enable all incoming alerts and configure them to classa.
```

Comment:
nit: this nit was already there
```suggestion
  // Enable all incoming alerts and configure them to class A.
```